### ISSUE TITLE
Does not work on IE at all

### ISSUE THREAD

**Issue by jcacchioni** [bfncs/jQuery-Youtube-Channel-Player#1]
Tried IE 7 / 8 / 9 - all I get is a blank page. Any suggestions to get this working across the board?

**Comment by bfncs** [comment]
Thanks for your feedback!
I didn't have the time to debug this in IE until now, but I'll definitely look into this.

**Comment by irabronson** [comment]
Hi.

Same here for IE 7-9; on Windows 7 Professional.

Not totally blank, just:

My Channel
To see what's going on, have a look at the source.
You can find the plugin and additional documentation in the Github repository.

In Windows, It's good on Firefox, Chrome, Safari.  Also good on Android.

**Comment by bfncs** [comment]
Ok, I found the time to look into this and it was caused by not requesting the data from the Youtube API with a [JSONP](http://en.wikipedia.org/wiki/JSONP) request. b7f20eed4bef830437ce43a55d7aa16864a89994 should have fixed this.

Thanks a lot for notifying me about the problem!

**Comment by irabronson** [comment]
Hi.  Thanks.  Sorry to bother, I don't see an index.html which is only an issue because also the demo link is now dead producing a 404.

Thank you.

**Comment by irabronson** [comment]
I was able to put it all together.  Works in IE 8 and IE 9 but not in IE 7... nothing displays in IE 7.

Thanks.

**Comment by bfncs** [comment]
Your comments are very much appreciated, no bothering at all!

I experimented with the github pages building engine and [Jekyll-Bootstrap](https://github.com/plusjade/jekyll-bootstrap) and deleteted it today, so probably that messed things up. After pushing to gh-pages, it is working again.

I'm going to look into the issue with IE7 in the next days.

**Comment by irabronson** [comment]
Awesome.  And, I was instantiating the player on the page differently, so now that I've referenced your demo, it DOES work in IE 7.  Sorry and thank you very much.

**Comment by bfncs** [comment]
Ok, nice that it works for you now. I would like to see, how you use it.

I'm closing down the issue then.

**Comment by irabronson** [comment]
> > > re: I would like to see, how you use it.

Here is the video page of the site: http://everythingwentblackmedia.com/videos

I also added some markup to be able to display the video title and number of views (the site is fully responsive and the views only display on the largest size).

I also included in the js, frameborder="0" allowtransparency="true", to be added to the iframe markup when it's instantiated on the page for IE8, 7.  Much rather use CSS but this always works better for those. 

Thanks.
